### DISCUSSION TITLE
Deactivate gender-equitable language in German translation

### DISCUSSION THREAD

**Discussion by KarlKeu00** [pretix/pretix#3954] [Q&A] [answered]
Is there somewhere to deactivate the gendering in the translation? In the context in which I use Pretix, we assume that "Teilnehmer" refers to both genders.

**Comment by raphaelm** [comment, marked as answer]
No.

**Reply by KarlKeu00** [reply]
Thank you for this detailed help or explanation as to why this gender language, which has not yet officially been formalised, is used at all.

**Reply by raphaelm** [reply]
The question did not ask for an explanation, it asked if there was a way to turn it off and there is not, so my answer was precise and comprehensive. Live with it, change it manually on your system (that's what the free in free software is all about) or use another system :)
